src/components/adjudications/AdjudicationForm.jsx

```diff
@@ -46,20 +50,55 @@ class AdjudicationForm extends React.Component {
     }
   }
 
+  getAwardConsderationEnumAction = (currAwardState, prevAwardState) => {
+    if (!isEqual(currAwardState, prevAwardState)) {
+      return currAwardState ? awardConsiderationEnum.INCREMENT : awardConsiderationEnum.DECREMENT;
```

Comment:
Do you think we should destructure all these references of awardConsiderationEnum? @livictoria